website/docs/docs/contributing/oss-expectations.md

```diff
@@ -0,0 +1,104 @@
+---
+title: "Expectations for OSS Contributors"
+---
+
+Whether it's a dbt package, a plugin, `dbt-core`, or this very documentation site, contributing to the open source code that supports the dbt ecosystem is a great way to level yourself up as a developer, and to give back to the community. The goal of this page is to help you understand what to expect when contributing to dbt open source software (OSS). While we can only speak for our own experience as open source maintainers, many of these guidelines apply when contributing to other open source projects, too.
+
+Have you seen things in other OSS projects that you quite like, and think we could learn from? Send us a message, or open a discussion--we're always excited to hear from you.
```

Comment:
> who? how?

good question! do you think this could be #community-strategy, or does this feel like conversation warrants its own dedicated swimlane?

> open a discussion

I think docs.getdbt.com is a good place

Reply:
I'd be happy with either #community-strategy or #dbt-core-development (or perhaps #package-ecosystem if their feedback specifically relates to those). As we start to expand our vision of "community" beyond just Slack, probably community-strategy is best if they're in Slack, or the docs.getdbt repo if they’re not 👍